composed_api.rst

```diff
@@ -287,20 +276,20 @@ Endpoint
 
 ::
 
-    GET <instance_url>/<instance_root>_ba/api/v1/aoi/add
+    GET <instance_url>/<instance_root>_ba/api/v2/aoi/add
 
 Request Parameters
 ^^^^^^^^^^^^^^^^^^
 
 +-------------------+-------------------------------------------------------+
 | Query parameter   | Value                                                 |
 +===================+=======================================================+
+| source            | *Required*. Your GRiD generated API key.              |
```

Comment:
I may be getting ahead of myself. Have we already changed how keys are generated? This says "GRiD generated" but last time around you just picked a key. Also `source` seems a strange name for a key. Maybe we should even take a step back and ask what purpose this is serving? Did you originally want to be able to track who was coming in via a particular application? This doesn't really do that, as the key is associated with the user, not an app, no? It may be an aside, but short of implementing OAuth for authorizing third-party applications, would providing the user a means of generating a revokable token (like Github personal access tokens) be doable. Still won't tell you what app the request came from.